src/rsz/test/pin_swap1.ok

```diff
@@ -10,431 +10,1314 @@
+sky130_fd_sc_hd__dlclkp_1/GATE ->
+sky130_fd_sc_hd__dlclkp_2/CLK ->
+sky130_fd_sc_hd__dlclkp_2/GATE ->
+sky130_fd_sc_hd__dlclkp_4/CLK ->
+sky130_fd_sc_hd__dlclkp_4/GATE ->
+sky130_fd_sc_hd__dlrbn_1/D ->
+sky130_fd_sc_hd__dlrbn_1/GATE_N ->
+sky130_fd_sc_hd__dlrbn_1/RESET_B ->
+sky130_fd_sc_hd__dlrbn_2/D ->
+sky130_fd_sc_hd__dlrbn_2/GATE_N ->
+sky130_fd_sc_hd__dlrbn_2/RESET_B ->
+sky130_fd_sc_hd__dlrbp_1/D ->
+sky130_fd_sc_hd__dlrbp_1/GATE ->
+sky130_fd_sc_hd__dlrbp_1/RESET_B ->
+sky130_fd_sc_hd__dlrbp_2/D ->
+sky130_fd_sc_hd__dlrbp_2/GATE ->
+sky130_fd_sc_hd__dlrbp_2/RESET_B ->
+sky130_fd_sc_hd__dlrtn_1/D ->
+sky130_fd_sc_hd__dlrtn_1/GATE_N ->
+sky130_fd_sc_hd__dlrtn_1/RESET_B ->
+sky130_fd_sc_hd__dlrtn_2/D ->
+sky130_fd_sc_hd__dlrtn_2/GATE_N ->
+sky130_fd_sc_hd__dlrtn_2/RESET_B ->
+sky130_fd_sc_hd__dlrtn_4/D ->
+sky130_fd_sc_hd__dlrtn_4/GATE_N ->
+sky130_fd_sc_hd__dlrtn_4/RESET_B ->
+sky130_fd_sc_hd__dlrtp_1/D ->
+sky130_fd_sc_hd__dlrtp_1/GATE ->
+sky130_fd_sc_hd__dlrtp_1/RESET_B ->
+sky130_fd_sc_hd__dlrtp_2/D ->
+sky130_fd_sc_hd__dlrtp_2/GATE ->
+sky130_fd_sc_hd__dlrtp_2/RESET_B ->
+sky130_fd_sc_hd__dlrtp_4/D ->
+sky130_fd_sc_hd__dlrtp_4/GATE ->
+sky130_fd_sc_hd__dlrtp_4/RESET_B ->
+sky130_fd_sc_hd__dlxbn_1/D ->
+sky130_fd_sc_hd__dlxbn_1/GATE_N ->
+sky130_fd_sc_hd__dlxbn_2/D ->
+sky130_fd_sc_hd__dlxbn_2/GATE_N ->
+sky130_fd_sc_hd__dlxbp_1/D ->
+sky130_fd_sc_hd__dlxbp_1/GATE ->
+sky130_fd_sc_hd__dlxtn_1/D ->
+sky130_fd_sc_hd__dlxtn_1/GATE_N ->
+sky130_fd_sc_hd__dlxtn_2/D ->
+sky130_fd_sc_hd__dlxtn_2/GATE_N ->
+sky130_fd_sc_hd__dlxtn_4/D ->
+sky130_fd_sc_hd__dlxtn_4/GATE_N ->
+sky130_fd_sc_hd__dlxtp_1/D ->
+sky130_fd_sc_hd__dlxtp_1/GATE ->
+sky130_fd_sc_hd__dlygate4sd1_1/A ->
+sky130_fd_sc_hd__dlygate4sd2_1/A ->
+sky130_fd_sc_hd__dlygate4sd3_1/A ->
+sky130_fd_sc_hd__dlymetal6s2s_1/A ->
+sky130_fd_sc_hd__dlymetal6s4s_1/A ->
+sky130_fd_sc_hd__dlymetal6s6s_1/A ->
+sky130_fd_sc_hd__ebufn_1/A ->
+sky130_fd_sc_hd__ebufn_1/TE_B ->
+sky130_fd_sc_hd__ebufn_2/A ->
+sky130_fd_sc_hd__ebufn_2/TE_B ->
+sky130_fd_sc_hd__ebufn_4/A ->
+sky130_fd_sc_hd__ebufn_4/TE_B ->
+sky130_fd_sc_hd__ebufn_8/A ->
+sky130_fd_sc_hd__ebufn_8/TE_B ->
+sky130_fd_sc_hd__edfxbp_1/CLK ->
+sky130_fd_sc_hd__edfxbp_1/D ->
+sky130_fd_sc_hd__edfxbp_1/DE ->
+sky130_fd_sc_hd__edfxtp_1/CLK ->
+sky130_fd_sc_hd__edfxtp_1/D ->
+sky130_fd_sc_hd__edfxtp_1/DE ->
+sky130_fd_sc_hd__einvn_0/A ->
+sky130_fd_sc_hd__einvn_0/TE_B ->
+sky130_fd_sc_hd__einvn_1/A ->
+sky130_fd_sc_hd__einvn_1/TE_B ->
+sky130_fd_sc_hd__einvn_2/A ->
+sky130_fd_sc_hd__einvn_2/TE_B ->
+sky130_fd_sc_hd__einvn_4/A ->
+sky130_fd_sc_hd__einvn_4/TE_B ->
+sky130_fd_sc_hd__einvn_8/A ->
+sky130_fd_sc_hd__einvn_8/TE_B ->
+sky130_fd_sc_hd__einvp_1/A ->
+sky130_fd_sc_hd__einvp_1/TE ->
+sky130_fd_sc_hd__einvp_2/A ->
+sky130_fd_sc_hd__einvp_2/TE ->
+sky130_fd_sc_hd__einvp_4/A ->
+sky130_fd_sc_hd__einvp_4/TE ->
+sky130_fd_sc_hd__einvp_8/A ->
+sky130_fd_sc_hd__einvp_8/TE ->
+sky130_fd_sc_hd__fa_1/A ->
+sky130_fd_sc_hd__fa_1/B ->
```

Comment:
Extra credit for a multi output gate :trophy:

Reply:
Done, but don't go asking for any Christmas presents this year.

![image](https://media.tenor.com/GMDGErB9z_QAAAAj/jorrbox-jorrparivar.gif)